Add `arg_exclusive()` (#1289)
r-lib/rlang: NEWS.md

## Features and bugfixes

> * `arg_exclusive()` checks that that arguments are not supplied
>   together (#1261).

Reviewer: ```suggestion
* `arg_exclusive()` checks that only one argument of a set is supplied (#1261).
```
